docs/ref/databases.txt

```diff
@@ -488,7 +488,10 @@ this entry are the four standard isolation levels:
 
 or ``None`` to use the server's configured isolation level. However, Django
 works best with and defaults to read committed rather than MySQL's default,
-repeatable read. Data loss is possible with repeatable read.
+repeatable read. Data loss is possible with repeatable read; in particular
```

Comment:
I think the semicolon would be better as a full stop. Multiple clause sentences are harder to read and harder to translate.

Reply:
Fixed; I have a bad semicolon habit.

Reply:
;)